api/auth.js

```diff
@@ -66,6 +66,12 @@ module.exports = {
       callback(null, isDbAdmin(userCtx));
     });
   },
+
+  isAdmin: function(userCtx) {
+    return hasRole(userCtx, '_admin') ||
+           hasRole(userCtx, 'national_admin');
+  },
```

Comment:
This shares most of its functionality with the `Session` service's `isAdmin` function, not sure if worth pulling it out to the shared library/if that's ready for use yet?

Reply:
Ideally we'd only have one implementation.  But from a quick look, it's not clear to me whether `Session` should be delegating to `Auth` to determine this, or the other way round.

Reply:
Well `Session` is client side and `auth` is server side so I don't think they'd necessarily delegate to each other, just share the same logic in some common utility function in the shared library